tool/teleport/testenv/test_server.go

```diff
@@ -393,6 +393,16 @@ func (p *cliModules) BuildType() string {
 	return "CLI"
 }
 
+// IsEnterpriseBuild returns if the binary was built with enterprise modules
+func (p *cliModules) IsEnterpriseBuild() bool {
+	return false
+}
+
+// IsOSSBuild returns if the binary was built without enterprise modules
```

Comment:
```suggestion
// IsOSSBuild returns false for [cliModules].
```

It is kinda of weird that this is neither. 🤷